network-api/networkapi/wagtailpages/pagemodels/buyersguide/editorial_content_index.py

```diff
@@ -0,0 +1,21 @@
+from wagtail.core import models as wagtail_models
+
+from networkapi.wagtailpages.pagemodels.mixin import foundation_metadata
+
+
+class BuyersGuideEditorialContentIndexPage(
+    foundation_metadata.FoundationMetadataPageMixin,
+    wagtail_models.Page,
+):
+    parent_page_types = ['wagtailpages.BuyersGuidePage']
+    subpage_types = ['wagtailpages.BuyersGuideArticlePage']
+    template = 'pages/buyersguide/editorial_content_index_page.html'
+
+    def get_context(self, request, *args, **kwargs):
+        context =  super().get_context(request, *args, **kwargs)
+        context["home_page"] = self.get_parent().specific
+        context["items"] = self.get_children()
```

Comment:
**Suggestion**

Little confused as to why we can't use just `self.get_descendants() ` here instead? But either way, it would be worth filtering for `.live()` pages to so drafts don't show up

Reply:
Good call! Thanks for catching this. 

The method is in anticipation of further changes. But you are right. I will make the change when it becomes necessary (YAGNI).